### PR TITLE
Bug 1145833 - Add link to the source and the currently deployed revision in mcMerge footer.

### PR DESCRIPTION
**Pull request by KWierso**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -152,5 +152,16 @@ <h1 id="title">mcMerge</h1>
     <noscript>
       You need Javascript to proceed.
     </noscript>
+
+    <hr />
+
+    <div id="footer">
+      <div id="source-link">
+        <a href="https://github.com/mozilla/mcMerge"> mcMerge Source </a>
```
Comment (edmorley) [inline index.html]: Trailing whitespace around link text here and below

File: index.html [hunk 2]
```diff
@@ -152,5 +152,16 @@ <h1 id="title">mcMerge</h1>
...
+      </div>
+      <div id="mcmerge-revision">
+        <a href="https://mcmerge.paas.allizom.org/deploy.txt"> Deployed revision </a>
```
Comment (edmorley) [inline index.html]: Would this be best as a relative link? Given the URL will be subject to change? I know the link will be broken locally, but I think that's perhaps more accurate locally anyway? Happy for you to make the final call on this :-)

Comment (KWierso) [inline index.html]: I switched it to a relative link and also added some CSS so it only shows up on the https://mcmerge.paas.allizom.org/ site, hidden everywhere else. 

It's a -moz prefixed rule, so it only hides in Firefox, but I think that'd be okay.

We'd have to update that rule should the hosting move somewhere else.

If you're okay with that, @edmorley I can merge this patch in.

**Comment by edmorley** [comment]
> I switched it to a relative link and also added some CSS so it only shows up on the https://mcmerge.paas.allizom.org/ site, hidden everywhere else.
> 
> It's a -moz prefixed rule, so it only hides in Firefox, but I think that'd be okay.
> 
> We'd have to update that rule should the hosting move somewhere else.
> 
> If you're okay with that, @edmorley I can merge this patch in.

Sounds fine to me - we can always iterate later if needs be :-)

**Pull request merged**